resources/user-agents/browsers/yabrowser/yabrowser-15-on.json

```diff
@@ -315,28 +315,37 @@
       },
       "children": [
         {
-          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#) AppleWebKit/* (KHTML* like Gecko) YaBrowser/#MAJORVER#.#MINORVER#* Safari/*",
+          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#) AppleWebKit/* (*khtml*like*gecko*) YaBrowser/#MAJORVER#.#MINORVER#* */*",
           "devices": {
             "iPad": "iPad",
-            "iPhone": "iPhone"
+            "iPhone": "iPhone",
+            "iPod": "iPod Touch"
           },
           "platforms": [
+            "iOS_A_7_0", "iOS_A_7_1",
+            "iOS_A_8_0", "iOS_A_8_1", "iOS_A_8_2", "iOS_A_8_3",
             "iOS_A_9_0", "iOS_A_9_1", "iOS_A_9_2", "iOS_A_9_3",
             "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2", "iOS_A_10_3",
             "iOS_A",
             "iOS_B",
+            "iOS_C_7_1", "iOS_C_7_0",
+            "iOS_C_8_3", "iOS_C_8_2", "iOS_C_8_1", "iOS_C_8_0",
             "iOS_C_9_3", "iOS_C_9_2", "iOS_C_9_1", "iOS_C_9_0",
             "iOS_C_10_3", "iOS_C_10_2", "iOS_C_10_1", "iOS_C_10_0",
             "iOS_C"
           ]
         },
         {
-          "match": "Mozilla/5.0 (#PLATFORM#) AppleWebKit/* (KHTML* like Gecko) YaBrowser/#MAJORVER#.#MINORVER#* Safari/*",
+          "match": "Mozilla/5.0 (#PLATFORM#) AppleWebKit/* (*khtml*like*gecko*) YaBrowser/#MAJORVER#.#MINORVER#* */*",
```

Comment:
There's a test for this, I saw some useragents that have `YaApp_iOS` in the spot where `Safari` is supposed to be.